test/serve/utils.dart

```diff
@@ -143,14 +143,18 @@ class DartTransformer extends Transformer {
 ///
 /// Returns the `pub serve` process.
 Future<TestProcess> startPubServe(
-    {Iterable<String> args, bool createWebDir: true, Compiler compiler}) async {
+    {Iterable<String> args,
+    bool createWebDir: true,
+    Compiler compiler,
+    bool forcePoll}) async {
```

Comment:
I am forwarding this through a couple methods though, and I want to handle nulls gracefully